pkgs/misc/drivers/argononed/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchFromGitLab, dtc, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "argononed";
+  version = "0.3.3";
+
+  src = fetchFromGitLab {
+    owner = "DarkElvenAngel";
+    repo = pname;
+    rev = "d50040115beece36a0259cef2248eff467286207";
+    sha256 = "sha256-3GBEzBh5VuuPh+iZeLBV/7oTX0Zqs8pcvfOXgDt68WQ=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  buildInputs = [ dtc ];
+
+  patchPhase = ''
```

```suggestion
  postPatch = ''
```
Use `postPatch` instead of `patchPhase` because it doesn't overwrite the hooks.
Also, patch phase comes before build phase. (So move it to before build block.)

---

Thank you!